syntax = "proto3";

option go_package = "pb/game";

message InterruptCurrentStatePbEvent
{
  uint64 actor_entity = 1;
  uint32 actor_action = 2;
  uint32 actor_state = 3;
}